curriculum/structure/superblocks/python-v9.json

```diff
@@ -14,6 +14,7 @@
             "lecture-understanding-functions-and-scope",
             "workshop-caesar-cipher",
             "lab-rpg-character",
+            "workshop-movie-ticket-booking-calculator",
```

Comment:
please move this to be right below `"lecture-numbers-and-mathematical-operations"`